appium/webdriver/extensions/ime.py

```diff
@@ -1,141 +0,0 @@
-#!/usr/bin/env python
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import warnings
-from typing import TYPE_CHECKING, List, cast
-
-from appium.protocols.webdriver.can_execute_commands import CanExecuteCommands
-
-from ..mobilecommand import MobileCommand as Command
-
-if TYPE_CHECKING:
-    from appium.webdriver.webdriver import WebDriver
-
-
-class IME(CanExecuteCommands):
-    @property
-    def available_ime_engines(self) -> List[str]:
-        """Get the available input methods for an Android device.
-
-        Package and activity are returned (e.g., ['com.android.inputmethod.latin/.LatinIME'])
-        Android only.
-
-        deprecated:: 2.0.0
-
-        Returns:
-            :obj:`list` of :obj:`str`: The available input methods for an Android device
-        """
-        warnings.warn(
-            'The "available_ime_engines" API is deprecated and will be removed in future versions. '
-            'Use "mobile: shell" extension instead',
```

Comment:
Should we add `mobile` command for IME handling? Because `mobile:shell` is behind a security flag

Reply:
I did not see any people using these so far as these APIs are very specific.
Eventually they could be called anyway using adb shell extension